src/main.rs

```diff
@@ -122,6 +124,14 @@ fn run_joshuto(args: Args) -> Result<(), JoshutoError> {
             }
         }
         QuitType::Force => {}
+        QuitType::Normal => {
+            if let Some(path) = args.choosefiles {
```

Comment:
Should be moved to `QuitType::ChooseFiles`.
Also you can move this into a separate function that returns `io::Result<()>`.
This way you don't need to use `unwrap()` and can just use `?` and handle the error here.